tests/admin_views/tests.py

```diff
@@ -864,6 +864,16 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    def test_submit_row(self):
+        """
+        #13875 - submit_row templatetag should pass whole context
+        """
+        change_url = reverse('admin:auth_user_change', args=[1])
```

Comment:
Shouldn't hardcode a primary key value -- that's why the test fails on non-sqlite databases.